Formula/automake.rb

```diff
@@ -15,20 +16,12 @@ class Automake < Formula
 
   depends_on "autoconf"
 
-  # Download more up-to-date config scripts.
-  resource "config" do
-    url "https://git.savannah.gnu.org/cgit/config.git/snapshot/config-0b5188819ba6091770064adf26360b204113317e.tar.gz"
-    sha256 "3dfb73df7d073129350b6896d62cabb6a70f479d3951f00144b408ba087bdbe8"
-    version "2020-08-17"
-  end
-
   def install
     on_macos do
       ENV["PERL"] = "/usr/bin/perl"
-    end
-
-    resource("config").stage do
-      cp Dir["config.*"], buildpath/"lib"
+      # TODO: Remove when https://github.com/Homebrew/brew/pull/10921
+      # lands in a tagged release of brew
```

Comment:
I think this whole PR has to wait for a tagged release, as any usage of autoconf 2.71 in other formulae relies on that being removed.

Reply:
I actually think it'll work fine in those cases as long as they don't run `automake`'s test for `autoconf`, but, yes, better to play it safe.

Reply:
It's a bit more than that. It affects any autoconf file that has a `AC_INIT` but no `AC_OUTPUT`.

Reply:
Aha, ok. I've managed to build some things just fine with `autoconf` + `automake` as in this PR while superenv still set `M4`, but I guess I was just lucky.